README.md

```diff
@@ -116,8 +116,8 @@ var transcriptStream = await client.getTranscriptObjectStream(job.id);
 
 ### Getting captions output
 
-Another way to retrieve your file is captions output. We currently only support .srt output. A readable stream of your srt output can be retrieved as such:
+Another way to retrieve your file is captions output. We support both .srt and .vtt outputs. A readable stream of your desired captions output can be retrieved as shown below. If your job was submitted as a multichannel file you will be required to provide the id of the channel you would like captioned.
 
 ```javascript
-var captionsStream = await client.getCaptions(job.id);
+var captionsStream = await client.getCaptions(job.id, CaptionTypes.SRT, null);
```

Comment:
I find the example confusing. As a customer, I would never provide null when there is a default value for this parameter.

Reply:
I want to show that the parameter exists but in most cases it will be null and error if soemthing is provided. Suggestions?

Reply:
Let's show another example, one with the param and one without. Add a comment to each to show why we are showing that example.